Next-formalizationFull.hs

```diff
@@ -83,20 +90,32 @@ good2 = EntityInstance{entRef = goodEntity, entInstName = "Auto", entInstType =
 product1 = RoleInstance{rolRef = roleProduct, rolInstName = "Product", actInstStyle = [productStyle], rolEnt = good1, rolInstCreate = (1, 1, 2016), rolInstDelete = emptyDateLast}
 product2 = RoleInstance{rolRef = roleProduct, rolInstName = "Product", actInstStyle = [productStyle], rolEnt = good2, rolInstCreate = (15, 2, 2016), rolInstDelete = emptyDateLast}
 
-delivery1a = EventInstance{evtRef = deliveryEvent, evtInstName = "Delivery", evtInstStyle = Regular, evtInstHasParty = True, evtInstParty = customer1, evtInstHasSubject = True, evtInstSubject = product1, evtInstHasAmount = True, evtInstHasValue = True, evtInstDirection  = DirOut, evtInstChangeOfOwnership = Always, evtInstTypeOfOwnership = TradeItem, evtInstPaymentRequired = Always, evtInstPlanned = [(1, 1, 2017), (5, 1, 2017), (10, 2, 2017)], evtInstActual = [(3, 1, 2017), (10, 2, 2017), (16, 3, 2017)], evtInstAdministrative = [], evtInstAmount = 5, evtInstValue = [100, 500]}
+delivery1a = EventInstance{evtRef = deliveryEvent, evtInstName = "delivery1a", evtInstStyle = Regular, evtInstHasParty = True, evtInstParty = customer1, evtInstHasSubject = True, evtInstSubject = product1, evtInstHasAmount = True, evtInstHasValue = True, evtInstDirection  = DirOut, evtInstChangeOfOwnership = Always, evtInstTypeOfOwnership = TradeItem, evtInstPaymentRequired = Never, evtInstPlanned = [(1, 1, 2017), (5, 1, 2017), (10, 2, 2017)], evtInstActual = [(3, 1, 2017), (10, 2, 2017), (16, 3, 2017)], evtInstAdministrative = [(1, 1, 2017), (5, 1, 2017), (10, 2, 2017)], evtInstAmount = 5, evtInstValue = [100, 500]}
```

Comment:
Based on Dennis' formalizations (https://www.dropbox.com/sh/wotluzcfcccm3rp/AAAOL7IqJk2OV96goCXHUXvPa?dl=0) the lifecycle (status) of an event instance will be determined using these dates , i.e. ToDo, Started, Done. For example, PlannedDate has three dates: the first one is for ToDo, the second one is for planned start, and the last one for the planned end date. However, I could not figure out why do we need ToDo date for ActualDate.